data/templates/redfish.1.0.0.eventdestination.json

```diff
@@ -0,0 +1,11 @@
+{
+    "@odata.context": "<%= basepath %>/$metadata#EventService/Members/Subscriptions/Members/$entity",
+    "@odata.id": "<%= url %>",
+    "@odata.type": "#EventService.1.0.0.EventDestination",
+    "Id": "<%= subscription.Id %>",
+    "Name": "Event Subscription Destination",
+    "Destination": "<%= subscription.Destination %>",
+    "EventTypes": <%- JSON.stringify(subscription.EventTypes) %>,
```

Comment:
Have we marked `<%-` as a valid delimiter (or is it another default?), or is this just a typo from `<%=` ?